public/js/app.js

```diff
@@ -1,45 +1,100 @@
-const displaySongs = function() {
-  const nowplayingImg = document.getElementById("nowplayingImg"),
-    nowplayingTitle = document.getElementById("nowplayingTitle"),
-    nowplayingInfo = document.getElementById("nowplayingInfo"),
-    lastPlayedDiv = document.getElementById("lastplayed"),
-    nowPlayingEndpoint = "https://35.211.223.26/nowplaying",
-    lastPlayedEndpoint = "https://35.211.223.26/lastplayed";
+const nowplayingImg = document.querySelector(".now-playing-img"),
+  nowplayingTitle = document.querySelector(".now-playing-title"),
+  nowplayingInfo = document.querySelector(".now-playing-info"),
+  nowPlayingArtist = document.querySelector(".now-playing-artist"),
+  recentlyPlayedDiv = document.querySelector(".recently-played-container"),
+  toggle = document.querySelector("#toggle"),
+  nowPlayingEndpoint = "http://35.211.223.26/nowplaying",
+  recentlyPlayedEndpoint = "http://35.211.223.26/recentlyplayed";
 
+const renderNowPlaying = song => {
+  const image = song.images.find(i => i.width === 640);
+
+  nowplayingImg.src = image[0].url;
```

Comment:
Update this like you did in `renderRecentlyPlayed()`. Also check if null. If possible, find some generic cover art to show in cases where the image is null. Maybe something like [this](https://orig00.deviantart.net/26aa/f/2011/185/f/9/no_cover_itunes_by_stainless2-d3kxnbe.png)

Reply:
copy that, I'll make those updates

Reply:
if image returns null then ill render that generic cover art